.github/workflows/pnp-input-api-build.yml

```diff
@@ -140,7 +144,7 @@ jobs:
       - name: Setup NuGet sources
         uses: extenda/actions/setup-nuget-sources@v0
         with:
-          config-file: src/nuget.config #check this in all of the APIs
```

Comment:
I guess the same should be added to `pnp-query-api-build.yml` workflow. The branch name suggests you meant to do this there as well but maybe forgot.